component-runtime-manager/src/main/resources/META-INF/services/org.talend.sdk.component.spi.component.ComponentMetadataEnricher

```diff
@@ -0,0 +1 @@
+org.talend.sdk.component.runtime.manager.extension.AfterVariableMetadataEnricher
```

Comment:
Maybe this spi should put in studio integration part, otherwise it will be in all envs...

Reply:
I agree that it'll be better. But right now not either studio-integration or component-runtime-di jars aren't available at the moment when org.talend.sdk.component.runtime.manager.reflect.ComponentMetadataService#ComponentMetadataService loads spi-s.

Reply:
Hum, strange that the SPI is not working in `org.talend.sdk.component.studio-integration`, the component-server is started from there...
Will double check this.

Reply:
I saw that "org.talend.sdk.component.studio-integration" comunicates with "componen-server" via WebSocket.

Reply:
Yes, using websocket but http is available also. This doesn't interfere in anything and process.
I will create a backlog task for investigating on spi loading in studio